File: pset1/Mario/less/mario.c
#include<stdio.h>
#include<cs50.h>
int main(void)
{
    int n;
    do
    {
    //printf("Height: ");
    //scanf("%d",&n);
    n=get_int("Height: ");
    }while(n<0 || n>23);
    for(int i=0;i<n;i++)
    {
        for(int j=0;j<n-i-1;j++)
        printf(" ");
        for(int k=0;k<i+2;k++)
        printf("#");
        printf("\n");
    }
}

File: pset6/Mario/less/mario.py
from cs50 import get_int
while True:
    n=get_int("Height: ")
    if n>8 or n<=0:
        continue
    else:
        for i in range(n):
            for w in range(n-i-1):
                print(" ",end='')
            for h in range(i+1):
                print("#",end='')
            print()
        break

